src/main/java/org/example/Console.java

```diff
@@ -0,0 +1,31 @@
+package org.example;
+
+import java.util.Scanner;
+
+public class Console {
+    private static final Scanner scanner = new Scanner(System.in);
+    public static void clear() {
+        try {
+            new ProcessBuilder("cmd", "/c", "cls").inheritIO().start().waitFor();
+        } catch (Exception ex) {
+            System.err.println("Console error: " + ex.getMessage());
+        }
+    }
+
+    public static void waitFor(final int milliseconds) {
+        try {
+            Thread.sleep(milliseconds);
+        } catch (InterruptedException intEX) {
+            intEX.getStackTrace();
```

Comment:
and you not handling the exemption if something happens, what will be the backup plan